File: Medium/2026-09-09-3871-Count-Commas-in-Range-II/solution.py
class Solution:
    def countCommas(self, n: int) -> int:
        if n < 1000:
            return 0
        
        count = 0
        for i in range(1000, n + 1, 1000):
            count += (i // 1000) * 1
        
        return count

File: Medium/2026-09-09-3871-Count-Commas-in-Range-II/solution.java
class Solution {
    public long countCommas(int n) {
        long count = 0;
        for (long i = 1; i <= n; i++) {
            if (i >= 1000) {
                count += (String.valueOf(i).length() - 1) / 3;
            }
        }
        return count;
    }
}